libraries/precisoUtils/bidUtils.js

```diff
@@ -101,3 +88,156 @@ function macroReplace(adm, cpm) {
   let replacedadm = replaceAuctionPrice(adm, cpm);
   return replacedadm;
 }
+
+export async function pid(sid) {
+  try {
+    const purl = 'https://ssp-usersync.mndtrk.com/getUUID?sharedId=' + sid;
+    const response = await fetch(purl);
+    var data = await response.json();
+    const dataMap = new Map(Object.entries(data));
+    const uuidValue = dataMap.get('UUID');
+    return uuidValue;
+  } catch (error) {
+    logInfo('Error in preciso pid fetch' + error);
+  }
+}
+
+function mapImpression(slot, bidderRequest) {
+  if (slot.mediaType === 'native' || deepAccess(slot, 'mediaTypes.native')) {
+    const imp = {
+      id: slot.bidId,
+      native: mapNative(slot),
+      bidfloor: slot.params.bidFloor,
```

Comment:
Sorry, not acceptable. You must first read the bid floor from the standard location as described in https://docs.prebid.org/dev-docs/bidder-adaptor.html . Specifically, [bidRequest.getFloor()](https://docs.prebid.org/dev-docs/modules/floors.html#bid-adapter-interface)

Reply:
@bretg Thanks for the updates. I will change bid floor read by using [bidRequest.getFloor()](https://docs.prebid.org/dev-docs/modules/floors.html#bid-adapter-interface)